qwik-graphql-tailwind/src/components/file-viewer/index.tsx

```diff
@@ -36,21 +39,27 @@ export const FileViewer = component$(() => {
       {
         owner,
         name,
-        expression: `${branch}:${path}`,
+        expression: `${globalStore.branch || pathBranch}:${path.replace(/\/+$/, '') || ''}`,
       },
       abortController
     );
+
     updateFile(store, response);
   });
 
   if (store.isLoading) {
-    return <div>Loading...</div>;
+    return <div />;
```

Comment:
Any reason for changing this?

Reply:
Yeah, using null, throws an error in the console. The intention was to just show a blank page while it loads, instead of showing a loading... text, because there is another component that will be showing that(in the layout). and we don't want to show a component(still loading its own data) without data in it(looks weird).